mcstatus/server.py

```diff
@@ -214,25 +239,36 @@ def __init__(self, host: str, port: int = 19132, timeout: float = 3):
         """Override init to add a default port for bedrock servers of 19132."""
         super().__init__(host, port, timeout=timeout)
 
+    @classmethod
+    def lookup(cls, address: str, timeout: float = 3) -> Self:
+        """Parse `address` parameter and return initialized object."""
+        host, port = address.split(":")
+        return cls(host, int(port), timeout=timeout)
+
+    @classmethod
+    async def async_lookup(cls, address: str, timeout: float = 3) -> Self:
+        """Just call `lookup` without any async, for implement all methods from abstract class."""
+        return cls.lookup(address, timeout)
```

Comment:
As stated in my previous comment, `async_lookup` makes no sense for bedrock, and shouldn't be present here since it has no actual async functionality.

Reply:
Deleted in 4ecf8832e0895740cac091345225ac45603e6ddf.